src/views/AddItem.jsx

```diff
@@ -1,3 +1,115 @@
-export function AddItem() {
-	return <p>Hello from the <code>/add-item</code> page!</p>
+import { useState } from 'react';
+import { addItem } from '../api/firebase';
+
+export function AddItem({ listToken }) {
+	const soon = 7;
+	const kindaSoon = 14;
+	const notSoon = 30;
```

Comment:
Depending on the company style, a actual literal constant is usually all caps. But again, depends on the company style guide (e.g. SOON, KINDA_SOON, NOT_SOON).

Some folks like to see what variables are just hardcoded literal constants, and which are changing (the ones below which useState).